components/layout/display_list/mod.rs

```diff
@@ -39,7 +39,6 @@ use webrender_api::{
 use wr::units::LayoutVector2D;
 
 use crate::context::{LayoutContext, ResolvedImage};
-use crate::display_list::conversions::ToWebRender;
```

Comment:
Maybe just make this one public instead of switching to a wildcard?

Reply:
Done, thanks!